pkg/cmd/kafka/topic/update/update.go

```diff
@@ -153,7 +154,6 @@ func NewUpdateTopicCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", "json", opts.localizer.MustLocalize("kafka.topic.common.flag.output.description"))
```

Comment:
Flag was removed but variables and validation is still present. Functionally it works but leaves dead code.